### PR TITLE
Add conv tests for BART and BERT

### PR DESCRIPTION
**Pull request by RissyRan**
# Description

Add conv tests for BART and BERT:
* Create helpers for commonly used commands
* Add helper config `get_flax_bart_conv_config` for BART conv tests - [source](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/jax/latest/flax-bart-wiki_summary.libsonnet)
* Add helper config `get_flax_bert_conv_config` for BERT conv tests - [source bert-mnli](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/jax/latest/flax-bert-glue_mnli.libsonnet), [source bert-mrpc](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/jax/latest/flax-bert-glue_mrpc.libsonnet)
* Other small changes, i.e. variable name change for ViT conv test

# Tests

Please describe the tests that you ran on Cloud VM to verify changes.

**Instruction and/or command lines to reproduce your tests:** ...
Upload to Airflow and test.

**List links for your tests (use go/shortn-gen for any internal link):** ...
* BART Conv test on v4-32
  * Airflow test pass: [link](https://screenshot.googleplex.com/nMSVzi2zaToMCJ2)
  * Conv metric [train_loss](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/jax/latest/flax-bart-wiki_summary.libsonnet#L65) can be found in post_process result and the alarm will be set up [dashboard](http://go/ml-auto-solutions-dashboard) once changes are merged and data is in BigQuery

```
[2024-01-09, 03:19:15 UTC] {logging_mixin.py:150} INFO - Test run rows: [TestRun(job_history=JobHistoryRow(uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2
e6bd57e6f3f1118c', timestamp=datetime.datetime(2024, 1, 9, 3, 19, 12, 153977), owner='Shiva S.', 
job_name='flax_bart_wiki_conv-v4-32', job_status=0), metric_history=
[MetricHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f11
18c', metric_key='train_time', metric_value=984.3043212890625), 
MetricHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f111
8c', metric_key='train_learning_rate', metric_value=1.8724799488722965e-08), 
MetricHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f111
8c', metric_key='train_loss', metric_value=0.9626200199127197), 
MetricHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f111
8c', metric_key='eval_loss', metric_value=0.9807181358337402), 
MetricHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f111
8c', metric_key='eval_rouge1', metric_value=1.0702999830245972), 
MetricHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f111
8c', metric_key='eval_rouge2', metric_value=0.17020000517368317), 
MetricHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f111
8c', metric_key='eval_rougeL', metric_value=1.0693000555038452), 
MetricHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f111
8c', metric_key='eval_rougeLsum', metric_value=1.065000057220459), 
MetricHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f111
8c', metric_key='eval_gen_len', metric_value=64.0)], metadata_history=
[MetadataHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3
f1118c', metadata_key='run_id', metadata_value='manual__2023-12-18T22:37:57.062465+00:00'), 
MetadataHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f
1118c', metadata_key='prev_start_date_success', metadata_value='2023-12-
14T00:18:08.979390+00:00'), 
MetadataHistoryRow(job_uuid='07f145ed31745093ee4bac590d4fe4b1ca1290354e5a9ea2e6bd57e6f3f
1118c', metadata_key='airflow_dag_run_link', 
metadata_value='https://7ef24502ca144d038c5754964af60450-dot-us-
central1.composer.googleusercontent.com/dags/flax_latest_supported/grid?
dag_run_id=manual__2023-12-
18T22%3A37%3A57.062465%2B00%3A00&task_id=flax_bart_wiki_conv-v4-
32.post_process.process_metrics')])]
```

* BERT - glue_mrpc conv test on v4-32
  * Airflow test pass: [link](https://screenshot.googleplex.com/Bh78R4H2EBKiALz)
  * Conv metric [eval_accuracy](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/jax/latest/flax-bert-glue_mrpc.libsonnet#L40) can be found in post_process result and the alarm will be set up [dashboard](http://go/ml-auto-solutions-dashboard)
  
```
[2024-01-09, 08:08:58 UTC] {logging_mixin.py:150} INFO - Test run rows: 
[TestRun(job_history=JobHistoryRow(uuid='8c7bb955cbdecbb7a9228370c9b17339a07ad4b76906fafc
dd446cab2dbce245', timestamp=datetime.datetime(2024, 1, 9, 8, 8, 57, 980901), owner='Shiva S.', 
job_name='flax_bert_mrpc_conv-v4-32', job_status=0), metric_history=
[MetricHistoryRow(job_uuid='8c7bb955cbdecbb7a9228370c9b17339a07ad4b76906fafcdd446cab2db
ce245', metric_key='eval_accuracy', metric_value=0.8504902124404907), 
MetricHistoryRow(job_uuid='8c7bb955cbdecbb7a9228370c9b17339a07ad4b76906fafcdd446cab2dbc
e245', metric_key='eval_f1', metric_value=0.8950086236000061)], metadata_history=
[MetadataHistoryRow(job_uuid='8c7bb955cbdecbb7a9228370c9b17339a07ad4b76906fafcdd446cab2
dbce245', metadata_key='run_id', metadata_value='manual__2023-12-18T22:37:57.062465+00:00'), 
MetadataHistoryRow(job_uuid='8c7bb955cbdecbb7a9228370c9b17339a07ad4b76906fafcdd446cab2
dbce245', metadata_key='prev_start_date_success', metadata_value='2023-12-
14T00:18:08.979390+00:00'), 
MetadataHistoryRow(job_uuid='8c7bb955cbdecbb7a9228370c9b17339a07ad4b76906fafcdd446cab2
dbce245', metadata_key='airflow_dag_run_link', 
metadata_value='https://7ef24502ca144d038c5754964af60450-dot-us-
central1.composer.googleusercontent.com/dags/flax_latest_supported/grid?
dag_run_id=manual__2023-12-
18T22%3A37%3A57.062465%2B00%3A00&task_id=flax_bert_mrpc_conv-v4-
32.post_process.process_metrics')])]
```

* BERT - glue_mnli conv test on v4-32
  * Airflow test pass: [link](https://screenshot.googleplex.com/83eVKBrWYugdSCg)
  * Conv metric [eval_accuracy](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/jax/latest/flax-bert-glue_mnli.libsonnet#L41) can be found in post_process result and the alarm will be set up [dashboard](http://go/ml-auto-solutions-dashboard)
  
```
[2024-01-09, 17:08:55 UTC] {logging_mixin.py:150} INFO - Test run rows: 
[TestRun(job_history=JobHistoryRow(uuid='7753404226ad754653fc8a03e7ff43b676e46a03afdbb21caf562557b7989
ca3', timestamp=datetime.datetime(2024, 1, 9, 17, 8, 36, 191985), owner='Shiva S.', job_name='flax_bert_mnli_conv-v4-
32', job_status=0), metric_history=
[MetricHistoryRow(job_uuid='7753404226ad754653fc8a03e7ff43b676e46a03afdbb21caf562557b7989ca3', 
metric_key='train_time', metric_value=56205.8671875), 
MetricHistoryRow(job_uuid='7753404226ad754653fc8a03e7ff43b676e46a03afdbb21caf562557b7989ca3', 
metric_key='train_learning_rate', metric_value=1.7059205958958046e-07), 
MetricHistoryRow(job_uuid='7753404226ad754653fc8a03e7ff43b676e46a03afdbb21caf562557b7989ca3', 
metric_key='train_loss', metric_value=0.32483866810798645), 
MetricHistoryRow(job_uuid='7753404226ad754653fc8a03e7ff43b676e46a03afdbb21caf562557b7989ca3', 
metric_key='eval_accuracy', metric_value=0.8415690064430237)], metadata_history=
[MetadataHistoryRow(job_uuid='7753404226ad754653fc8a03e7ff43b676e46a03afdbb21caf562557b7989ca3', 
metadata_key='run_id', metadata_value='manual__2023-12-18T22:37:57.062465+00:00'), 
MetadataHistoryRow(job_uuid='7753404226ad754653fc8a03e7ff43b676e46a03afdbb21caf562557b7989ca3', 
metadata_key='prev_start_date_success', metadata_value='2023-12-14T00:18:08.979390+00:00'), 
MetadataHistoryRow(job_uuid='7753404226ad754653fc8a03e7ff43b676e46a03afdbb21caf562557b7989ca3', 
metadata_key='airflow_dag_run_link', metadata_value='https://7ef24502ca144d038c5754964af60450-dot-us-
central1.composer.googleusercontent.com/dags/flax_latest_supported/grid?dag_run_id=manual__2023-12-
18T22%3A37%3A57.062465%2B00%3A00&task_id=flax_bert_mnli_conv-v4-32.post_process.process_metrics')])]

```

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

**Review by sshahrokhi — APPROVED** [review]
Thanks!
